roles/kerberos/README.md

```diff
@@ -9,6 +12,7 @@ An Ansible role to configure Ansible Tower nodes to manage Windows systems.
 |`krb_realms`|yes|N/A|List of Dictionaries|Used for storing the realm(domain) name and its domain controllers.  A single domain controller can be specified if that is the only available option, but more than one is preferable. See example usage below.|
 |`krb_default_realm`|no|undefined|string|If a default realm(domain) is required to be specified this can be set.  Otherwise it remains unset in the krb5.conf.|
 |`krb_dns_lookup_realm`|yes|"false"|string|Whether or not to lookup DNS via realm.|
+|`dns_lookup_kdc`|no|"true"|string|Indicate whether DNS SRV records should be used to locate the KDCs and other servers for a realm.|
```

Comment:
The variable is correctly called `krb_dns_lookup_kdc`.

Reply:
I've updated the documentation, and added a file about setting up a windows AD domain. Had referred to my old Lab notes from College and put them in a markdown.